model/textparse/openmetricsparse.go

```diff
@@ -94,13 +95,15 @@ type OpenMetricsParser struct {
 	exemplarVal   float64
 	exemplarTs    int64
 	hasExemplarTs bool
+	skipCT        bool
```

Comment:
Could we add a comment explaining why `skipCT` exists? This could help someone (including us in the future) that is reading this code and wondering why would we ever want to skip anything 🤔

Reply:
Added something, but there's possibly room for improvement